SweetPea to English/gpt3prompt_derived_factors.txt

```diff
@@ -282,3 +284,118 @@ Text:
+
+Code:
+num      = factor("digit",  [1, 2, 3, 4, 5, 6, 7, 8, 9, 10])
+color      = factor("Color",  ["red", "green", "blue", "brown"])
+word       = factor("Word", ["red", "green", "blue", "brown"])
+
+def is_e(num):
+    return (int(num) % 2) == 0
+
+def is_o(num):
+    return not is_e(num)
+
+o_level = derived_level("odd", within_trial(is_e, [num]),3)
+e_level = derived_level("even", within_trial(is_o, [num]),2)
+
+par_factor = factor("parity", [
+    o_level,
+    e_level
+])
+
+Text:
+The parity factor depends on the regular factor digit has two levels: odd and even. A trial is considered even if the digit is divisible by 2, otherwise, it is considered odd. In our experiment in the factor parity the ratio between odd and even is 3 to 2.
+
+Code:
+num      = factor("number",  range(0,20))
+mot      = factor("motion",  ["up", "down"])
+shp       = factor("shape", ["circle", "square", "triangle"])
+
+def is_larg_10(num):
+    return int(num) > 10
+
+def is_small_eq_10(num):
+    return not is_larg_10(num)
+
+large_level = derived_level("large", within_trial(is_larg_10, [num]),1)
+small_level = derived_level("small", within_trial(is_small_eq_10, [num]),weight=5)
+
+mag_factor = factor("magnitude", [
+    large_level,
+    small_level
+])
+
+Text:
+The magnitude factor depends on the regular factor number has two levels: large and small. A trial is considered large if the number is larger than 10, otherwise, it is considered small. In the magnitude factor the level "small" appeared 5 times as often as the level "large".
+
+Code:
+target = factor("number", ['1', '2', '3', '4', '5'])
+color      = factor("Color",  ["red", "green"])
+word       = factor("Word", ["red", "green"])
+task       = factor("Task", ["color naming", "word reading"])
+
+def is_lrg(num_factor):
+    return int(num_factor[0]) < int(num_factor[1])
+
+def is_eq(num_factor):
+    return int(num_factor[0]) == int(num_factor[1])
+
+def is_sml(num_factor):
+    return not is_lrg(num_factor) and not is_eq(num_factor)
+
+numtransfac = factor("number transition", [
+    derived_level("increase",
+                 transition(is_lrg, [target]), 3),
+    derived_level("unchanged",
+                  transition(is_eq, [target]), 2),
+    derived_level("decrease",
+                 transition(is_sml, [target]))
+])
+
+Text:
+The number transition factor depends on the transition of the number factor and has three levels: increase, decrease, or unchanged. If the number on the current trial is larger than the previous one, then the trial is considered an increase. If the number on the current trial is equal to the number on the previous trial, then the trial is considered unchanged. Otherwise, the trial is considered a decrease. In our experiment, the increase transition occurs three time as often as the decrease transition. The unchanged transition occurs twice as often.
```

Comment:
```suggestion
The number transition factor depends on the transition of the number factor and has three levels: increase, decrease, or unchanged. If the number on the current trial is larger than the previous one, then the trial is considered an increase. If the number on the current trial is equal to the number on the previous trial, then the trial is considered unchanged. Otherwise, the trial is considered a decrease. The increase transition occurs three times as often as the decrease transition, and the unchanged transition occurs twice as often as the decrease transition.
```